ts/image-occlusion/shapes/from-cloze.ts

```diff
@@ -9,11 +9,14 @@ import type { Shape, ShapeOrShapes } from "./base";
 import { Ellipse } from "./ellipse";
 import { Point, Polygon } from "./polygon";
 import { Rectangle } from "./rectangle";
+import { Text } from "./text";
 
 /** Given a cloze field with text like the following, extract the shapes from it:
  * {{c1::image-occlusion:rect:left=10.0:top=20:width=30:height=10:fill=#ffe34d}}
  */
-export function extractShapesFromClozedField(clozeStr: string): ShapeOrShapes[] {
+export function extractShapesFromClozedField(
+    clozeStr: string,
+): ShapeOrShapes[] {
```

Comment:
Not a big deal, but noting for the future: these changes in formatting make the review a little more difficult - I'm guessing you may have formatted with prettier? You should be able to configure VSCode to use dprint instead once the dprint extension (in .vscode.dist's recommendations) is installed.

Reply:
Yes, I have prettier set as the default. I'll change that. I made sure to run `ninja format` before committing, but apparently running prettier before dprint sometimes results in dprint behaving differently.

Reply:
Some formatters try to preserve manual formatting, so they'll e.g. wrap long lines, but won't unwrap short lines that could be on a single line. Personally I think that's a bad idea as it leads to problems like this, but it's not something we can change unfortunately :-) IIRC, Apple's Swift formatter behaves like this too.